mopidy_gmusic/commands.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import print_function
+from oauth2client.client import OAuth2WebServerFlow
+import gmusicapi
+from mopidy import commands
```

Comment:
We usually have an empty line in-between each import block. Future imports are one block. stdlib are one block. Each third-party library is one block. The current project is one block. In other words, an empty line in-between each of the four import lines here.

In the future, we'll hopefully have this automated with Black and isort, so it doesn't have be a topic in reviews.